### PR TITLE
Fix: enable scf_ene_thr for EXX outer loop

### LINKED ISSUE #4979: `scf_ene_thr` is not fully compatible with EXX SCF
### Describe the Code Quality Issue

I've test HSE calculation for Si by using:
- only `scf_thr 1e-7`
- only `scf_ene_thr 1e-7` (with `scf_thr` default, or set to `scf_thr -1`, this setting seems to be viewed as default)
- `scf_thr 1e-7` and `scf_thr_ene 1e-8`

And it can be seen that `scf_ene_thr` only effective in the XC-SCF part, but no use in EXX part. So that EXX part need to be converged when reach `scf_thr` but not `scf_ene_thr`

[Si-HSE.tar.gz](https://github.com/user-attachments/files/16671294/Si-HSE.tar.gz)

Seems `scf_ene_thr ` is not fully compatible

### Additional Context

output for `scf_ene_thr 1e-7`  along with `scf_thr -1`

```
                              ABACUS v3.7.3

               Atomic-orbital Based Ab-initio Computation at UStc                    

                     Website: http://abacus.ustc.edu.cn/                             
               Documentation: https://abacus.deepmodeling.com/                       
                  Repository: https://github.com/abacusmodeling/abacus-develop       
                              https://github.com/deepmodeling/abacus-develop         
                      Commit: 53afa6aa0 (Thu Aug 15 17:21:36 2024 +0800)

 Tue Aug 20 16:00:52 2024
 MAKE THE DIR         : OUT.ABACUS/
 RUNNING WITH DEVICE  : CPU / Intel(R) Xeon(R) Platinum 8358 CPU @ 2.60GHz
 dft_functional readin is: hse
 dft_functional in pseudopot file is: PBE
 Please make sure this is what you need
 UNIFORM GRID DIM        : 72 * 72 * 72
 UNIFORM GRID DIM(BIG)   : 24 * 24 * 24
 DONE(0.201922   SEC) : SETUP UNITCELL
 DONE(0.205999   SEC) : INIT K-POINTS
 ---------------------------------------------------------
 Self-consistent calculations for electrons
 ---------------------------------------------------------
 SPIN    KPOINTS         PROCESSORS  NBASE       
 1       63              16          104         
 ---------------------------------------------------------
 Use Systematically Improvable Atomic bases
 ---------------------------------------------------------
 ELEMENT ORBITALS        NBASE       NATOM       XC          
 Si      2s2p1d-7au      13          8           
 ---------------------------------------------------------
 Initial plane wave basis and FFT box
 ---------------------------------------------------------
 DONE(0.217344   SEC) : INIT PLANEWAVE
(base) [2201110432@wm2-login01 ene_thr_only_2]$ tail -f abacus.out 
 ---------------------------------------------------------
 ELEMENT ORBITALS        NBASE       NATOM       XC          
 Si      2s2p1d-7au      13          8           
 ---------------------------------------------------------
 Initial plane wave basis and FFT box
 ---------------------------------------------------------
 DONE(0.217344   SEC) : INIT PLANEWAVE
 -------------------------------------------
 SELF-CONSISTENT : 
 -------------------------------------------
 START CHARGE      : atomic
 DONE(30.7883    SEC) : INIT SCF
 * * * * * *
 << Start SCF iteration.
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.43015319e+02   0.00000000e+00   1.5428e-01   0.56
 GE2     -8.41805233e+02   1.21008588e+00   1.0188e-01   0.43
 GE3     -8.42963821e+02  -1.15858786e+00   1.0287e-01   0.42
 GE4     -8.44567128e+02  -1.60330663e+00   2.3495e-02   0.42
 GE5     -8.44633512e+02  -6.63843650e-02   7.3991e-03   0.42
 GE6     -8.44641177e+02  -7.66461706e-03   2.5937e-03   0.42
 GE7     -8.44641952e+02  -7.75647317e-04   7.1739e-04   0.42
 GE8     -8.44642019e+02  -6.64694207e-05   6.2501e-04   0.42
 GE9     -8.44642077e+02  -5.83025022e-05   1.4754e-04   0.42
 GE10    -8.44642080e+02  -3.46586101e-06   6.4050e-05   0.41
 GE11    -8.44642081e+02  -5.72485669e-07   1.3563e-05   0.42
 Updating EXX and rerun SCF     3.281e+00 (s)
 GE0     -8.44642081e+02  -1.93041173e-08   3.4287e-06   3.71
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.43481698e+02   0.00000000e+00   3.3946e-02   0.52
 GE2     -8.43948116e+02  -4.66418120e-01   4.1404e-02   0.45
 GE3     -8.44020903e+02  -7.27869869e-02   1.8085e-02   0.45
 GE4     -8.44022151e+02  -1.24846866e-03   1.3155e-02   0.45
 GE5     -8.44016238e+02   5.91313896e-03   7.8023e-03   0.45
 GE6     -8.44017842e+02  -1.60361718e-03   1.4680e-03   0.45
 GE7     -8.44018025e+02  -1.83483320e-04   2.6990e-04   0.45
 GE8     -8.44018028e+02  -2.73644994e-06   6.8899e-05   0.47
 GE9     -8.44018029e+02  -4.21731188e-07   2.4185e-05   0.46
 Updating EXX and rerun SCF     2.924e+00 (s)
 GE0     -8.44018029e+02  -6.42301141e-08   6.3315e-06   3.38
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44026642e+02   0.00000000e+00   1.7130e-03   0.51
 GE2     -8.44026624e+02   1.73503768e-05   5.5085e-04   0.45
 GE3     -8.44026641e+02  -1.71814793e-05   6.3360e-04   0.44
 GE4     -8.44026713e+02  -7.10623370e-05   1.4207e-04   0.45
 GE5     -8.44026716e+02  -3.46161952e-06   5.3008e-05   0.47
 GE6     -8.44026716e+02  -4.47268187e-07   1.3281e-05   0.47
 Updating EXX and rerun SCF     2.889e+00 (s)
 GE0     -8.44026716e+02  -2.71379258e-08   5.1287e-06   3.37
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027056e+02   0.00000000e+00   4.2180e-04   0.49
 GE2     -8.44027054e+02   1.32566537e-06   1.5053e-04   0.45
 GE3     -8.44027057e+02  -2.19789995e-06   1.6045e-04   0.45
 GE4     -8.44027061e+02  -4.62440455e-06   3.4582e-05   0.45
 GE5     -8.44027061e+02  -2.07415771e-07   1.3084e-05   0.47
 Updating EXX and rerun SCF     3.008e+00 (s)
 GE0     -8.44027061e+02  -2.73338846e-08   3.3535e-06   3.46
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027083e+02   0.00000000e+00   1.1487e-04   0.48
 GE2     -8.44027083e+02   1.13073643e-07   4.3975e-05   0.45
 GE3     -8.44027083e+02  -2.44960686e-07   4.4785e-05   0.45
 GE4     -8.44027083e+02  -3.36709912e-07   9.4723e-06   0.47
 Updating EXX and rerun SCF     2.942e+00 (s)
 GE0     -8.44027083e+02  -1.57324854e-08   3.7589e-06   3.39
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   3.3709e-05   0.49
 Updating EXX and rerun SCF     2.905e+00 (s)
 GE0     -8.44027085e+02   1.24596737e-08   1.2998e-05   3.35
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   1.7466e-05   0.48
 Updating EXX and rerun SCF     2.903e+00 (s)
 GE0     -8.44027085e+02   2.02135325e-08   2.3920e-05   3.35
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   3.7160e-06   0.48
 Updating EXX and rerun SCF     2.911e+00 (s)
 GE0     -8.44027085e+02   1.49642147e-09   3.9772e-06   3.36
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   5.3262e-06   0.48
 Updating EXX and rerun SCF     2.872e+00 (s)
 GE0     -8.44027085e+02   2.70533358e-09   9.4390e-06   3.32
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   7.9325e-07   0.49
 Updating EXX and rerun SCF     2.848e+00 (s)
 GE0     -8.44027085e+02   2.28151345e-10   9.5246e-07   3.30
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   1.9565e-06   0.47
 Updating EXX and rerun SCF     2.803e+00 (s)
 GE0     -8.44027085e+02   5.70765060e-10   3.5047e-06   3.25
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   3.6717e-07   0.49
 Updating EXX and rerun SCF     2.874e+00 (s)
 GE0     -8.44027085e+02  -4.72930670e-10   4.4207e-07   3.32
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   6.9418e-07   0.48
 Updating EXX and rerun SCF     2.994e+00 (s)
 GE0     -8.44027085e+02  -4.63939960e-10   1.2273e-06   3.44
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   1.6108e-07   0.48
 Updating EXX and rerun SCF     2.891e+00 (s)
 GE0     -8.44027085e+02   1.14694388e-09   1.8373e-07   3.34
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   2.3711e-07   0.48
 Updating EXX and rerun SCF     2.942e+00 (s)
 GE0     -8.44027085e+02   9.41801019e-10   4.1462e-07   3.39
 ITER       ETOT/eV          EDIFF/eV         DRHO     TIME/s
 GE1     -8.44027085e+02   0.00000000e+00   6.2737e-08   0.49
 >> Leave SCF iteration.
 * * * * * *
----------------------------------------------------------------
 TOTAL-STRESS (KBAR)                                            
----------------------------------------------------------------
       161.9618350592      -208.9796408569        66.4470578649 
      -208.5655862971       348.1221207605        40.5711056280 
        66.5100531408        38.1480592241       154.4498376356 
----------------------------------------------------------------
 TOTAL-PRESSURE: 221.511264 KBAR

TIME STATISTICS
--------------------------------------------------------------------------------
      CLASS_NAME                  NAME            TIME/s  CALLS   AVG/s  PER/%  
--------------------------------------------------------------------------------
                       total                      143.03 9        15.89  100.00 
 Driver                reading                    0.06   1        0.06   0.05   
 Input_Conv            Convert                    0.00   1        0.00   0.00   
 Driver                driver_line                142.97 1        142.97 99.95  
 UnitCell              check_tau                  0.00   1        0.00   0.00   
 ESolver_KS_LCAO       before_all_runners         22.78  1        22.78  15.92  
 PW_Basis_Sup          setuptransform             0.01   1        0.01   0.01   
 PW_Basis_Sup          distributeg                0.00   1        0.00   0.00   
 mymath                heapsort                   0.00   3        0.00   0.00   
 PW_Basis_K            setuptransform             0.01   1        0.01   0.00   
 PW_Basis_K            distributeg                0.00   1        0.00   0.00   
 PW_Basis              setup_struc_factor         0.00   1        0.00   0.00   
 NOrbital_Lm           extra_uniform              6.50   1875     0.00   4.55   
 Mathzone_Add1         SplineD2                   0.01   1875     0.00   0.01   
 Mathzone_Add1         Cubic_Spline_Interpolation 0.08   1875     0.00   0.05   
 Mathzone_Add1         Uni_Deriv_Phi              6.29   1875     0.00   4.40   
 Exx_LRI               init                       22.56  1        22.56  15.77  
 Matrix_Orbs21         init                       1.95   2        0.97   1.36   
 ORB_gaunt_table       init_Gaunt_CH              0.15   3        0.05   0.11   
 ORB_gaunt_table       Calc_Gaunt_CH              0.08   82305    0.00   0.05   
 ORB_gaunt_table       init_Gaunt                 1.03   3        0.34   0.72   
 ORB_gaunt_table       Get_Gaunt_SH               4.81   3635313  0.00   3.37   
 Matrix_Orbs21         init_radial                0.00   2        0.00   0.00   
 Matrix_Orbs21         init_radial_table          15.92  2        7.96   11.13  
 Center2_Orb           cal_ST_Phi12_R             12.35  3439     0.00   8.63   
 LRI_CV                set_orbitals               15.61  1        15.61  10.91  
 Matrix_Orbs11         init                       0.01   1        0.01   0.01   
 Matrix_Orbs11         init_radial                0.00   1        0.00   0.00   
 Matrix_Orbs11         init_radial_table          4.24   1        4.24   2.96   
 ppcell_vl             init_vloc                  0.00   1        0.00   0.00   
 Ions                  opt_ions                   120.03 1        120.03 83.91  
 ESolver_KS_LCAO       runner                     81.42  1        81.42  56.92  
 ESolver_KS_LCAO       before_scf                 7.86   1        7.86   5.50   
 ESolver_KS_LCAO       beforesolver               0.31   1        0.31   0.22   
 ESolver_KS_LCAO       set_matrix_grid            0.30   1        0.30   0.21   
 atom_arrange          search                     0.00   1        0.00   0.00   
 Grid_Technique        init                       0.29   1        0.29   0.21   
 Grid_BigCell          grid_expansion_index       0.00   2        0.00   0.00   
 Record_adj            for_2d                     0.00   1        0.00   0.00   
 Grid_Driver           Find_atom                  0.00   560      0.00   0.00   
 LCAO_domain           grid_prepare               0.00   1        0.00   0.00   
 Veff                  initialize_HR              0.00   1        0.00   0.00   
 OverlapNew            initialize_SR              0.00   1        0.00   0.00   
 EkineticNew           initialize_HR              0.00   1        0.00   0.00   
 NonlocalNew           initialize_HR              0.00   1        0.00   0.00   
 Exx_LRI               cal_exx_ions               7.52   1        7.52   5.26   
 LRI_CV                cal_datas                  1.29   3        0.43   0.90   
 Charge                set_rho_core               0.00   1        0.00   0.00   
 Charge                atomic_rho                 0.02   1        0.02   0.01   
 PW_Basis_Sup          recip2real                 0.17   315      0.00   0.12   
 PW_Basis_Sup          gathers_scatterp           0.07   315      0.00   0.05   
 Potential             init_pot                   0.01   1        0.01   0.01   
 Potential             update_from_charge         1.81   62       0.03   1.27   
 Potential             cal_fixed_v                0.00   1        0.00   0.00   
 PotLocal              cal_fixed_v                0.00   1        0.00   0.00   
 Potential             cal_v_eff                  1.81   62       0.03   1.26   
 H_Hartree_pw          v_hartree                  0.08   62       0.00   0.06   
 PW_Basis_Sup          real2recip                 0.18   329      0.00   0.12   
 PW_Basis_Sup          gatherp_scatters           0.09   329      0.00   0.06   
 PotXC                 cal_v_eff                  1.72   62       0.03   1.20   
 XC_Functional         v_xc                       16.87  38       0.44   11.79  
 Potential             interpolate_vrs            0.00   62       0.00   0.00   
 H_Ewald_pw            compute_ewald              0.00   1        0.00   0.00   
 Charge_Mixing         init_mixing                0.00   16       0.00   0.00   
 HSolverLCAO           solve                      25.82  61       0.42   18.05  
 HamiltLCAO            updateHk                   4.13   3843     0.00   2.89   
 OperatorLCAO          init                       1.80   15372    0.00   1.26   
 Veff                  contributeHR               1.73   61       0.03   1.21   
 Gint_interface        cal_gint                   2.79   123      0.02   1.95   
 Gint_interface        cal_gint_vlocal            1.43   61       0.02   1.00   
 Gint_Tools            cal_psir_ylm               0.55   35136    0.00   0.38   
 Gint_k                transfer_pvpR              0.29   61       0.00   0.20   
 OverlapNew            calculate_SR               0.00   1        0.00   0.00   
 OverlapNew            contributeHk               0.23   3843     0.00   0.16   
 EkineticNew           contributeHR               0.00   61       0.00   0.00   
 EkineticNew           calculate_HR               0.00   1        0.00   0.00   
 NonlocalNew           contributeHR               0.06   61       0.00   0.04   
 NonlocalNew           calculate_HR               0.00   1        0.00   0.00   
 OperatorLCAO          contributeHk               1.81   3843     0.00   1.27   
 HSolverLCAO           hamiltSolvePsiK            18.38  3843     0.00   12.85  
 DiagoElpa             elpa_solve                 16.90  3843     0.00   11.82  
 ElecStateLCAO         psiToRho                   3.30   61       0.05   2.31   
 elecstate             cal_dm                     0.54   62       0.01   0.38   
 psiMulPsiMpi          pdgemm                     0.53   3906     0.00   0.37   
 DensityMatrix         cal_DMR                    0.96   62       0.02   0.67   
 Gint                  transfer_DMR               0.19   61       0.00   0.13   
 Gint_interface        cal_gint_rho               1.30   61       0.02   0.91   
 Charge_Mixing         get_drho                   0.00   61       0.00   0.00   
 Charge                mix_rho                    0.03   60       0.00   0.02   
 Charge                Broyden_mixing             0.02   60       0.00   0.01   
 RI_2D_Comm            split_m2D_ktoR             1.09   15       0.07   0.76   
 Exx_LRI               cal_exx_elec               42.89  15       2.86   29.99  
 RI_2D_Comm            add_HexxR                  0.28   49       0.01   0.20   
 XC_Functional         v_xc_libxc                 1.62   49       0.03   1.13   
 ESolver_KS_LCAO       after_scf                  1.70   1        1.70   1.19   
 ModuleIO              save_sparse                1.13   1        1.13   0.79   
 ESolver_KS_LCAO       out_deepks_labels          0.00   1        0.00   0.00   
 LCAO_Deepks_Interface out_deepks_labels          0.00   1        0.00   0.00   
 HamiltLCAO            updateSk                   0.00   63       0.00   0.00   
 ESolver_KS_LCAO       cal_force                  38.61  1        38.61  26.99  
 Force_Stress_LCAO     getForceStress             38.61  1        38.61  26.99  
 Forces                cal_force_loc              0.00   1        0.00   0.00   
 Forces                cal_force_ew               0.00   1        0.00   0.00   
 Forces                cal_force_cc               0.00   1        0.00   0.00   
 Forces                cal_force_scc              0.00   1        0.00   0.00   
 Stress_Func           stress_loc                 0.02   1        0.02   0.01   
 Stress_Func           stress_har                 0.00   1        0.00   0.00   
 Stress_Func           stress_ewa                 0.00   1        0.00   0.00   
 Stress_Func           stress_cc                  0.00   1        0.00   0.00   
 Stress_Func           stress_gga                 0.02   1        0.02   0.01   
 Force_LCAO            ftable                     0.10   1        0.10   0.07   
 Force_LCAO            allocate                   0.02   1        0.02   0.02   
 LCAO_domain           build_ST_new               0.00   2        0.00   0.00   
 LCAO_domain           vnl_mu_new                 0.02   1        0.02   0.01   
 Force_LCAO            cal_fedm                   0.01   1        0.01   0.01   
 Force_LCAO            cal_ftvnl_dphi             0.00   1        0.00   0.00   
 Force_LCAO            cal_fvl_dphi               0.05   1        0.05   0.03   
 Gint_interface        cal_gint_force             0.05   1        0.05   0.03   
 Gint_Tools            cal_dpsir_ylm              0.01   288      0.00   0.01   
 Gint_Tools            cal_dpsirr_ylm             0.00   288      0.00   0.00   
 Force_LCAO            cal_fvnl_dbeta             0.01   1        0.01   0.01   
 Exx_LRI               cal_exx_force              13.35  1        13.35  9.34   
 Exx_LRI               cal_exx_stress             25.11  1        25.11  17.56  
 ESolver_KS_LCAO       cal_stress                 0.00   1        0.00   0.00   
 ESolver_KS_LCAO       after_all_runners          0.02   1        0.02   0.01   
 ModuleIO              write_istate_info          0.02   1        0.02   0.01   
--------------------------------------------------------------------------------


 START  Time  : Tue Aug 20 16:00:52 2024
 FINISH Time  : Tue Aug 20 16:03:16 2024
 TOTAL  Time  : 144
 SEE INFORMATION IN : OUT.ABACUS/
```

### Task list for Issue attackers (only for developers)

- [ ] Identify the specific code file or section with the code quality issue.
- [ ] Investigate the issue and determine the root cause.
- [ ] Research best practices and potential solutions for the identified issue.
- [ ] Refactor the code to improve code quality, following the suggested solution.
- [ ] Ensure the refactored code adheres to the project's coding standards.
- [ ] Test the refactored code to ensure it functions as expected.
- [ ] Update any relevant documentation, if necessary.
- [ ] Submit a pull request with the refactored code and a description of the changes made.

### PR DESCRIPTION
**Pull request by maki49**
fix #4979

### CODE REVIEW

File: source/module_ri/Exx_LRI_interface.hpp [hunk 1]
```diff
@@ -158,45 +161,50 @@ bool Exx_LRI_Interface<T, Tdata>::exx_after_converge(
...
+
+                std::cout << " Updating EXX " << std::flush;
+                timeval t_start;       gettimeofday(&t_start, nullptr);
+
+                const bool flag_restart = (this->two_level_step == 0) ? true : false;
+                this->mix_DMk_2D.mix(dm.get_DMK_vector(), flag_restart);
+
+                // GlobalC::exx_lcao.cal_exx_elec(p_esolver->LOC, p_esolver->LOWF.wfc_k_grid);
+                const std::vector<std::map<int, std::map<std::pair<int, std::array<int, 3>>, RI::Tensor<Tdata>>>>
+                    Ds = GlobalV::GAMMA_ONLY_LOCAL
```
Comment (mohanchen) [inline source/module_ri/Exx_LRI_interface.hpp]: avoid using GlobalV

File: source/module_ri/Exx_LRI_interface.hpp [hunk 2]
```diff
@@ -158,45 +161,50 @@ bool Exx_LRI_Interface<T, Tdata>::exx_after_converge(
...
+                    ? RI_2D_Comm::split_m2D_ktoR<Tdata>(*this->exx_ptr->p_kv, this->mix_DMk_2D.get_DMk_gamma_out(), *dm.get_paraV_pointer(), GlobalV::NSPIN)
```
Comment (mohanchen) [inline source/module_ri/Exx_LRI_interface.hpp]: avoid using GlobalV

File: source/module_ri/Exx_LRI_interface.hpp [hunk 3]
```diff
@@ -158,45 +161,50 @@ bool Exx_LRI_Interface<T, Tdata>::exx_after_converge(
...
-            iter = 0;
-            this->two_level_step++;
-            
-            timeval t_end;       gettimeofday(&t_end, nullptr);
-            std::cout << "and rerun SCF\t"
-                << std::setprecision(3) << std::setiosflags(std::ios::scientific)
-                << (double)(t_end.tv_sec-t_start.tv_sec) + (double)(t_end.tv_usec-t_start.tv_usec)/1000000.0 
-                << std::defaultfloat << " (s)" << std::endl;
-            return false;
+                timeval t_end;       gettimeofday(&t_end, nullptr);
```
Comment (mohanchen) [inline source/module_ri/Exx_LRI_interface.hpp:201]: initialize the variable t_end

File: source/module_ri/Exx_LRI_interface.hpp [hunk 4]
```diff
@@ -158,45 +161,50 @@ bool Exx_LRI_Interface<T, Tdata>::exx_after_converge(
...
-        {
-            restart_reset();
-            return true;
-        }
         else
-        {
-            // update exx and redo scf
-            if (this->two_level_step == 0)
+        { // has separate_loop case
+            double ediff = std::abs(etot - etot_last_outer_loop) * ModuleBase::Ry_to_eV;
```
Comment (mohanchen) [inline source/module_ri/Exx_LRI_interface.hpp]: add const before double ediff is OK?

File: source/module_ri/Exx_LRI_interface.hpp (hunk 3 above)
Comment (maki49) [reply source/module_ri/Exx_LRI_interface.hpp:201]: This is a standard struct in `sys/time.h` that cannot be directly assigned.

**Review by mohanchen — APPROVED** [review]

**Pull request merged**